### PR TITLE
send survey by week, month or quarter

### PR DESCRIPTION
**Pull request by lamngockhuong**
https://dev.framgia.com/redmine/issues/161748

### CODE REVIEW

File: app/Console/Commands/ResendReminderEmailCommand.php [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+                switch ($type) {
+                    case config('settings.reminder.week'):
+                        $nextTime->addWeek();
+                        break;
+                    case config('settings.reminder.month'):
+                        $nextTime->addMonth();
+                        break;
+                    case config('settings.reminder.quarter'):
+                        $nextTime->addQuarter();
+                        break;
```
Comment (at-diephq) [inline app/Console/Commands/ResendReminderEmailCommand.php]: add case default

File: resources/assets/js/component.js [hunk 2]
```diff
@@ -405,6 +412,48 @@ $(document).ready(function() {
...
+        switch ($type) {
+            case 'week':
+                nextTime.setDate(nextTime.getDate() + 7);
+                break;
+            case 'month':
+                nextTime.setMonth(nextTime.getMonth() + 1);
+                break;
+            case 'quarter':
+                nextTime.setMonth(nextTime.getMonth() + 3);
+                break;
```
Comment (at-diephq) [inline resources/assets/js/component.js]: case default

File: app/Console/Commands/ResendReminderEmailCommand.php [hunk 3]
```diff
@@ -0,0 +1,114 @@
...
+     *
+     * @return mixed
+     */
+    public function handle()
+    {
+        $surveys = $this->surveyRepository
+            ->with('user', 'settings')
+            ->whereHas(
+                'settings',
+                function ($query) {
```
Comment (trangkarol) [inline app/Console/Commands/ResendReminderEmailCommand.php]: Convention: `->whereHas( 'settings',  function ($query) {`

File: app/Console/Commands/ResendReminderEmailCommand.php (hunk 1 above)
Comment (lamngockhuong) [reply app/Console/Commands/ResendReminderEmailCommand.php]: Đã fix :D

File: resources/assets/js/component.js (hunk 2 above)
Comment (lamngockhuong) [reply resources/assets/js/component.js]: Đã fix :D

**Pull request merged**